### PR TITLE
Dlc 928 synchronizer webpack

### PR DESCRIPTION
**Pull request by barmintor**
Gets synchronizer running as a webpacked bundle
Updates Ableplayer  and Hls.j releases
Cleans up unused assets
Various Bootstrap fixups
Reduces number of selectable palettes to three in advance of plan to consolidate on monochrome and blue, with darkmode support

### CODE REVIEW

**Review by elohanlon — APPROVED** [review]

**Comment by elohanlon** [comment]
This was a lot of work!  Thanks for doing this, @barmintor.

**Comment by elohanlon** [comment]
Two minor notes about synchronizer things I noticed while running locally, and these definitely don't have to be taken care of for this PR:

1) Might be good to add a bit of margin/padding between AblePlayer and the righthand synchronized index/transcript.  Seems a little tight.

2) I'm seeing a both vertical and horizontal scrollbars in the `#right-sidebar` area, when you only want a vertical one.  I think this comes from the use of the `overflow-auto` class on `#right-sidebar`.  Ideally it shouldn't be producing the horizontal scrollbar at all (since it's `auto` and the content isn't wider than the container when `overflow-auto` is removed), but a quick fix would be to set overflow-y to `auto` and overflow-x to `hidden` if that doesn't cause any issues.  Then maybe later, further restructuring of some of the elements/css to eliminate a `hidden` fix would be even better.  @er-k may have some good thoughts about this too during his PR review.

File: app/views/shared/synchronizer.html.erb
Comment (er-k) [inline app/views/shared/synchronizer.html.erb:13]: to fix overflow and gap, change class to: class="d-flex flex-column flex-md-row mx-lg-n2"

File: app/views/shared/synchronizer.html.erb
Comment (er-k) [inline app/views/shared/synchronizer.html.erb:14]: to fix overflow and gap, change class to: class="w-100 vh-sm-50 h-md-auto px-lg-2"

File: app/views/shared/synchronizer.html.erb
Comment (er-k) [inline app/views/shared/synchronizer.html.erb:20]: to fix overflow and gap, change class to: class="vh-md-100 min-vh-50 p-0 w-100 overflow-auto px-lg-2"

**Review by er-k — CHANGES_REQUESTED** [review]
outstanding work! 
i agree with @elohanlon's comments re player/transcript overflow and see some minor modifications we can make to smooth these out. have and will continue to ticket these individually after this pr is merged in.
a few suggestions inline.
(tl;dr: adding _mx-lg-n2_ class to _#working-area_ and _px-lg-2_ class to _#right-sidebar_ and _#left-sidebar_ will fix the overflow and gutter issue)

**Comment by barmintor** [comment]
The requested style changes are in, and some improved error handling in Hls shim. Deployed to staging (except for the last commit, which removes some console debugging). Will merge when @er-k approves.

**Review by er-k — APPROVED** [review]

File: app/views/shared/synchronizer.html.erb
Comment (elohanlon) [inline app/views/shared/synchronizer.html.erb:20]:
```suggestion
      <div id="right-sidebar" class="vh-md-100 min-vh-50 p-0 w-100 px-lg-2 py-sm-3" style="max-height:80vh;overflow-y:auto;overflow-x:hidden;">
```
May be good to add a little padding to the top of the right sidebar so that its content is closer in alignment to the top of the AblePlayer box.

Comment (elohanlon) [inline app/views/shared/synchronizer.html.erb:20]:
```suggestion
      <div id="right-sidebar" class="vh-md-100 min-vh-50 p-0 w-100 px-lg-2 pt-sm-3" style="max-height:80vh;overflow-y:auto;overflow-x:hidden;">
```
May be good to add a little padding to the top of the right sidebar so that its content is closer in alignment to the top of the AblePlayer box.

**Pull request merged**
